jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/IPC.java

```diff
@@ -98,18 +103,22 @@ public class IPC {
 
 	static {
 		Properties props = VM.internalGetProperties();
-		String osName = props.getProperty("os.name"); //$NON-NLS-1$
+		String osName = props.getProperty("os.name").toLowerCase(); //$NON-NLS-1$
```

Comment:
Why does this need to use `toLowerCase()`? I think `equals()` could be used instead of `equalsIgnoreCase()` for z/OS.

Reply:
This matches other references to the system property `os.name` such as
https://github.com/eclipse-openj9/openj9/blob/2095c8e18d6d1230c1db39ae6ecfd6064131f988/jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/ManagementUtils.java#L92-L97

Reply:
I don't think there's a need to convert to lower-case nor do case-insensitive comparisons (I'd call it an "anti-pattern"): we know what the various O/S strings will be. It's just a waste of time (and in some cases space) to have the code be case-insensitive. I'd be happy if you (or anyone) offers to fix the case-insensitive sequences.

Reply:
Removed `toLowerCase()` and case-insensitive comparisons, and defer the changes of other places to a separate PR.